config/locales/es.yml

```diff
@@ -2490,6 +2490,13 @@ es:
         edit:
           county: Condado
           title: Selecciona el condado donde viviste el 31 de diciembre de %{filing_year}
+      nj_household_rent_own:
+        edit:
+          helper_description: Si usted es dueño de una casa móvil que está ubicada en un parque de casas móviles, se le considera un inquilino.
+          homeowner: Propietario de vivienda en %{filing_year}
+          neither: Tampoco
+          tenant: Inquilino en %{filing_year}
+          title: "¿Cuál te describe mejor?"
```

Comment:
[dust] double checking this was added as a line item to our spanish review ticket